src/dolphin/workflows/single.py

```diff
@@ -252,6 +258,13 @@ def run_wrapped_phase_single(
             in_no_pad_rows.start,
             in_no_pad_cols.start,
         )
+        with h5py.File(cslc_filename, "a") as hf:
+            nrows, ncols = cur_comp_slc.shape
+            row_start = in_no_pad_rows.start
```

Comment:
I now see that the typing is a little off in `single.py`, but just pointing out that `in_no_pad_rows` / `in_no_pad_cols` are already the `slice`s that you want here